homeassistant/components/sma/entities.py

```diff
@@ -0,0 +1,776 @@
+"""Entity Descriptions for the sma integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    ELECTRIC_CURRENT_AMPERE,
+    ELECTRIC_CURRENT_MILLIAMPERE,
+    ELECTRIC_POTENTIAL_VOLT,
+    ENERGY_KILO_WATT_HOUR,
+    ENERGY_WATT_HOUR,
+    FREQUENCY_HERTZ,
+    PERCENTAGE,
+    POWER_VOLT_AMPERE,
+    POWER_WATT,
+    TEMP_CELSIUS,
+)
+from homeassistant.helpers.entity import EntityCategory
+
+SENSOR_ENTITIES: dict[str, SensorEntityDescription] = {
```

Comment:
Please move the sensor entity descriptions to the sensor platform. We want the descriptions in each corresponding platform as they are platform details.

Reply:
Fixed with [d68fdb6](https://github.com/home-assistant/core/pull/58707/commits/d68fdb6b66057413979a6133b41cbb64f81939f8)